.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,75 @@
+name: Feature Request
+description: Ask for new feature or feature enhancement
+title: "Feature request Title"
+labels: ['type: enhancement', triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+
+  - type: textarea
+    id: feature-summary
+    attributes:
+      label: Enhancement summary
+      value: "please detail your feature enhancement"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: component
+    attributes:
+      label: Which component of AVD is impacted
+      multiple: true
+      options:
+        - eos_designs
+        - eos_cli_config_gen
+        - cvp_configlet_upload
+        - dhcp_provisioner
+        - eos_config_deploy_cvp
+        - eos_config_deploy_eapi
+        - eos_snapshot
+        - eos_validate_state
+    validations:
+      required: true
+
+  - type: textarea
+    id: feature-use-case
+    attributes:
+      label: Use case example
+      value: "please detail your use case"
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-solution
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: alternative-solution
+    attributes:
+      label: Describe alternatives you've considered
```

Comment:
update to: "Describe alternatives you have considered"

Reply:
Fixed